reana_server/rest/workflows.py

```diff
@@ -401,10 +401,15 @@ def create_workflow(user):  # noqa
         workflow_dict["operational_options"] = validate_operational_options(
             workflow_engine, reana_spec_file.get("inputs", {}).get("options", {})
         )
+        workspace_root_path = reana_spec_file.get("workspace", {}).get(
+            "workspace_root_path", None
```

Comment:
Do we have to call it `workspace_root_path`? Isn't it a bit redundant having `workspace` at the top level?
IMO we could simplify it and use just `path` or `root_path`. What do you think?

```yaml
...
workspace:
  path: /eos/home-s/simko/myworkflows
...
```

BTW, is it definite to have `workspace` at the top level? I thought we were discussing it [here](https://github.com/reanahub/reana-client/issues/545).

PS. For `get()`, you don't need to specify `None` as it's the default value.

Reply:
cc/ @tiborsimko @audrium

Reply:
I used `workspace_root_path` as it is not the full `workspace_path`, the full path would also have the `workflow_uuid`. So I agree that `root_path` would be a good simplification

Reply:
Looks good, +1 for `workspace_path`.
Also, @marcdiazsan,  for `get()`, you don't need to specify `None` as @mvidalgarcia mentioned

Reply:
Yes this one can be closed as is included into the other PR